contracts/handlers/fee/V2/DynamicGenericFeeHandlerEVMV2.sol

```diff
@@ -29,7 +29,16 @@ contract DynamicGenericFeeHandlerEVMV2 is DynamicFeeHandlerV2 {
         uint256 maxFee = uint256(bytes32(depositData[:32]));
         uint256 desintationCoinPrice = twapOracle.getPrice(destinationNativeCoinWrap[destinationDomainID]);
         if (desintationCoinPrice == 0) revert IncorrectPrice();
-        uint256 txCost = destinationGasPrice[destinationDomainID] * maxFee * desintationCoinPrice / 1e18;
+        Fee memory destFeeConfig = destinationFee[destinationDomainID];
+
+        uint256 txCost = destFeeConfig.gasPrice * maxFee * desintationCoinPrice / 1e18;
+        if(destFeeConfig.feeType == 0x01) {
```

Comment:
To avoid magic numbers in the code, could we have those declared as constants, or even as Enum?
```suggestion
        if(destFeeConfig.feeType == FEE_TYPE_FIXED) {
```